plugins/woocommerce-admin/client/customize-store/assembler-hub/resizable-frame.jsx

```diff
@@ -119,6 +152,18 @@ function ResizableFrame( {
 
 		setIsOversized( updatedWidth > defaultSize.width );
 
+		if ( updatedWidth >= tabletStyles.width ) {
+			setDeviceType( 'Desktop' );
+		}
+
+		if ( updatedWidth <= tabletStyles.width ) {
+			setDeviceType( 'Tablet' );
+		}
+
+		if ( updatedWidth <= mobileStyles.width ) {
+			setDeviceType( 'Mobile' );
+		}
+
```

Comment:
Given that we removed the support for `ResizableFrame,` are these changes still necessary?

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/50070/commits/aeb1c2cbb4749f37cf48423d8316b6afe2903169